src/pages/robot/_refactor/account/actionBar.tsx

```diff
@@ -1,30 +1,31 @@
-import React, { Component } from 'react';
+/* eslint-disable */
```

Comment:
this not good, for what?

Reply:
I think it was already there, I just moved it to top of file I guess
I can remove it